test/bigtest/tests/loan-actions-history-test.js

```diff
@@ -18,16 +20,17 @@ describe('loans actions history', () => {
     });
   });
 
+  let openLoan;
+
   beforeEach(async function () {
-    const loan = this.server.create('loan', {
+    openLoan = this.server.create('loan', {
       status: { name: 'Open' },
       loanPolicyId: 'test'
     });
 
-    this.server.createList('loanactions', 5, { loan: { ...loan.attrs } });
-
-    this.server.createList('request', requestsAmount, { itemId: loan.itemId });
-    this.visit(`/users/${loan.userId}/loans/view/${loan.id}`);
+    this.server.createList('loanactions', 5, { loan: { ...openLoan.attrs } });
+    this.server.createList('request', requestsAmount, { itemId: openLoan.itemId });
+    this.visit(`/users/${openLoan.userId}/loans/view/${openLoan.id}`);
```

Comment:
See that in one [place](https://github.com/folio-org/ui-users/pull/1080/files#diff-0c6e69ff7433118d17d13645a1ec126bR110) you have an empty line before `this.visit`. Please make things consistent.